Makefile

```diff
@@ -1,3 +1,4 @@
 update_metadata:
-	(cd libPhoneNumberTests && ./metadataGenerator) && \
-		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh)
+	cd libPhoneNumberTests && ./metadataGenerator
+	./jsonToPlist.swift < libPhoneNumberTests/generatedJSON/PhoneNumberMetaData.json > libPhoneNumber/NBPhoneNumberMetaData.plist
```

Comment:
This doesn't seem correct. Maybe we should rename the Test json as `PhoneNumberMetaDataTest.json` to be explicit

Reply:
Sure, I can conditionalize the plist name just like the old constant was conditionalized under `TESTING`. (I was the one who told Martin we don't bother with any of the testing stuff, but it was handy for checking that I didn't break anything, so I think it's worth keeping in some form.)